File: exams/frmDBConnection.vb
Public Class frmDBConnection
    Private Sub btnSave_Click(sender As Object, e As EventArgs) Handles btnSave.Click
        My.Settings.dbName = txtDBName.Text
        My.Settings.host = txtDbHost.Text
        My.Settings.userName = txtUserName.Text
        My.Settings.passWord = txtPassword.Text
        My.Settings.dPport = txtDbPort.Text
        My.Settings.API = txtAPI.Text
        My.Settings.APIUserName = txtAPIUserName.Text
        My.Settings.Sender = txtSender.Text
        My.Settings.Save()
        success("The Connection Was Succesfuly Saved")
        Me.Close()
    End Sub

    Private Sub frmDBConnection_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        txtDBName.Text = My.Settings.dbName
        txtDbHost.Text = My.Settings.host
        txtUserName.Text = My.Settings.userName
        txtPassword.Text = My.Settings.passWord
        txtDbPort.Text = My.Settings.dPport
        txtAPI.Text = My.Settings.API
        txtSender.Text = My.Settings.Sender
        txtAPIUserName.Text = My.Settings.APIUserName
    End Sub

    Private Sub defaultConnString_Click(sender As Object, e As EventArgs) Handles defaultConnString.Click
        Me.txtDBName.Text = "schoolss"
        Me.txtDbHost.Text = "localhost"
        Me.txtPassword.Text = String.Empty
        Me.txtUserName.Text = "root"
        Me.txtDbPort.Text = "7474"
    End Sub
End Class

File: exams/frmDBConnection.cs
using System;

namespace exams
{
    public partial class frmDBConnection
    {
        public frmDBConnection()
        {
            InitializeComponent();
            _btnSave.Name = "btnSave";
            _defaultConnString.Name = "defaultConnString";
        }

        private void btnSave_Click(object sender, EventArgs e)
        {
            My.MySettingsProperty.Settings.dbName = txtDBName.Text;
            My.MySettingsProperty.Settings.host = txtDbHost.Text;
            My.MySettingsProperty.Settings.userName = txtUserName.Text;
            My.MySettingsProperty.Settings.passWord = txtPassword.Text;
            My.MySettingsProperty.Settings.dPport = txtDbPort.Text;
            My.MySettingsProperty.Settings.API = txtAPI.Text;
            My.MySettingsProperty.Settings.APIUserName = txtAPIUserName.Text;
            My.MySettingsProperty.Settings.Sender = txtSender.Text;
            My.MySettingsProperty.Settings.Save();
            publicSubsNFunctions.success("The Connection Was Succesfuly Saved");
            Close();
        }

        private void frmDBConnection_Load(object sender, EventArgs e)
        {
            txtDBName.Text = My.MySettingsProperty.Settings.dbName;
            txtDbHost.Text = My.MySettingsProperty.Settings.host;
            txtUserName.Text = My.MySettingsProperty.Settings.userName;
            txtPassword.Text = My.MySettingsProperty.Settings.passWord;
            txtDbPort.Text = My.MySettingsProperty.Settings.dPport;
            txtAPI.Text = My.MySettingsProperty.Settings.API;
            txtSender.Text = My.MySettingsProperty.Settings.Sender;
            txtAPIUserName.Text = My.MySettingsProperty.Settings.APIUserName;
        }

        private void defaultConnString_Click(object sender, EventArgs e)
        {
            txtDBName.Text = "schoolss";
            txtDbHost.Text = "localhost";
            txtPassword.Text = string.Empty;
            txtUserName.Text = "root";
            txtDbPort.Text = "7474";
        }
    }
}